### PR TITLE
DM-44513:update the default value

### PR DESCRIPTION
**Pull request by wguanicedew**
## Checklist

- [ ] ran Jenkins
- [ ] added a release note for user-visible changes to `doc/changes`

### CODE REVIEW

**Review by MichelleGower — COMMENTED** [review]
Why are the values left in but just commented out?  Also, numberOfRetries default is now defined by ctrl_bps.

**Comment by wguanicedew** [comment]
> Why are the values left in but just commented out? Also, numberOfRetries default is now defined by ctrl_bps.

These items are removed.

File: doc/changes/DM-44513.misc.rst [hunk 1]
```diff
@@ -0,0 +1 @@
+change the defult memory to 4000 and the default memoryMultipler to 1.5
```
Comment (MichelleGower) [inline doc/changes/DM-44513.misc.rst:1]: This description needs to be changed.

**Review by MichelleGower — APPROVED** [review]
Thanks for removing the commented out settings.  Changes look good.  I would update the description in the doc/changes file before merging.

File: doc/changes/DM-44513.misc.rst (hunk 1 above)
Comment (wguanicedew) [reply doc/changes/DM-44513.misc.rst]: I changed the description.

**Pull request merged**